polygon-digital-carbon/networkConfig/localhost.json

```diff
@@ -95,5 +95,9 @@
   "CMARK": {
     "address": "0xeee3abdd638e219261e061c06c0798fd5c05b5d3",
     "startBlock": "67036935"
+  },
+  "TVER": {
+    "address": "0xB95A8C12D0F49e7388De4CF9a17EEE28d734D7A1",
```

Comment:
assuming you've already tested locally, so this is working even if not all lowercase

Reply:
haven't gotten the tests working locally yet, problem with my local node setup...

Reply:
when you create the fork you most likely need to specify `--host 0.0.0.0`. so:

```anvil --fork-url $(POLYGON_URL) --host 0.0.0.0 --no-storage-caching```

not as crucial if you always update the local graph start blocks to the fork block but in docker-compose.yml adding `ETHEREUM_REORG_THRESHOLD: 10` to environment to not overload the rpc

Probably most important though is to delete the data/ folder in the graph-node/docker. i delete it before every new local graph node spin up as it can be a frequent source of errors